### ISSUE TITLE
How to obtain the policy(logits) rather than only the actions from self.actor.step()

### ISSUE THREAD

**Issue by fuxianh** [ShangtongZhang/DeepRL#104]
Hello, I want to obtain the Q network output, name it as logists, it is a distribution over actions and can be seen as Q(s, a_i). But as seen in DQN_agent.py, line 103, transitions = self.actor.step(), I could not get it. Is there any convenient way to get this information? Thank you so much. I am looking forward to your reply.

**Comment by fuxianh** [comment]
Specificially, I want the behavior policy (q values over differrent actions, from the actor) stored in the transition buffer too. Is there any way to do this?

**Comment by fuxianh** [comment]
I find the solution in DQNActor._transition. No bother.
